src/environments/route.ts

```diff
@@ -153,6 +153,9 @@ export const USER_ROUTES = {
       username
     )}/learning-objects/${objectId}/materials`;
   },
+  GET_CHILDREN(learningObjectID: string){
+    return `${environment.apiURL}/learning-objects/${learningObjectID}/children/summary`;
```

Comment:
we should wrap params in encodeURIComponent to ensure we don't accidentally send up improper requests.

```suggestion
    return `${environment.apiURL}/learning-objects/${encodeURIComponent(learningObjectID)}/children/summary`;
```